server/services/TokenService.js

```diff
@@ -0,0 +1,18 @@
+const Token = require("../models/Token");
+const TokenRepository = require("../repositories/TokenRepository");
+
+class TokenService{
+
+  constructor(){
+    this.tokenRepository = new TokenRepository();
+  }
+
+  async getByUUID(uuid){
+    const tokenObject = await this.tokenRepository.getByUUID(uuid);
+    const token = new Token(tokenObject.id);
```

Comment:
If we decide to check input types on the wallet, then we should maybe check it here too (or in the Token model constructor).

Reply:
In my opinion, I tend to demand every object to check the input from outside. Because this makes wrongdoing be found at the earlier place, because if I don't care no matter what kind of wrong/inappropriate inputs given to me, and deal with them, do some logic, and return them back or pass them to other modules, then it would becomes difficult for us to debug and find the source of the error.

But, sometimes I am lazy on this, sometimes it's tedious to do it at every place.... :D